L2-counter/deploy/deploy.ts

```diff
@@ -0,0 +1,52 @@
+import { utils, Wallet } from 'zksync-web3';
+import * as ethers from 'ethers';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { Deployer } from '@matterlabs/hardhat-zksync-deploy';
+
+const GOVERNANCE_ADDRESS = '0x3C4Ac9AfadB3F345744248Ec97DC3051c63d7c00';
+
+// An example of a deploy script that will deploy and call a simple contract.
```

Comment:
"will be deployed" or "we/you will deploy"

Reply:
fixed in c524d3a1470771290f717b27c219f20ecab10e72